ui/app/templates/components/select-dropdown.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="field">
+  <label
+    for="select-dropdown-{{this.elementId}}"
+    class="is-label"
+    data-test-select-label
+    >
+    {{dropdownLabel}}
+  </label>
+  <div class="control {{if isInline "select is-inline-block"}}">
+    <div class="select {{if isFullwidth "is-fullwidth"}}">
+    <select
+      class="select"
+      id="select-dropdown-{{this.elementId}}"
+      name="selectedItem"
```

Comment:
Do we need `name` for anything?

Reply:
nope, i'm just in the habit of including one. good catch!